LDAR_Sim/src/virtual_world/fugitive_emission.py

```diff
@@ -59,6 +59,7 @@ def __init__(
         self._tagging_rep_delay: int = 0
         self._nrd: int = nrd
         self._repair_date: date = None
+        self._simulation_sd = start_date  # Start date
```

Comment:
Question: Why do we need to keep the simulation start date?

Reply:
Good catch, this was a relic from before where the programs were initialized before the processes were spawned. This will be removed.